iana/entID.go

```diff
@@ -0,0 +1,4 @@
+package iana
+
+// EntIDCiscoSystems is IANA Enterprise ID for Cisco Systems
+const EntIDCiscoSystems = 0x9
```

Comment:
Can you define a type like in all the rest of the definitions on the IANA package?

Reply:
This comment still stands.